app/views/posts/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Show a list of all related posts of user <%=params[:user_id]%><h1>
\ No newline at end of file
+<%= stylesheet_link_tag('posts_index') %>
+<section class="post_index_section">
+    <article class="user_card">
+        <img src="https://pm1.narvii.com/7258/5520799cf0539b408bd8abee0a14d3a492ee5107r1-753-753v2_hq.jpg" alt="User photo">
+        <div class="user_info">
+            <h2><%= @user.name %></h2>
+            <h4>Number of post: <%= @user.posts_counter %></h4>
+        </div>
+    </article>
+    <% @user.posts.each do |post| %>
+    <%= render "post_card", post: post %>
+    <% post.comments.each do |comment| %>
+    <ul class="comments_list">
+        <%= render "comment_card", comment: comment %>
+    </ul>
+    <%end%>
+    <%end%>
```

Comment:
- [ ] From the project requirements, here you are expected to render the 5 recent comments for each post using the custom method built in the previous milestone, `recent_comments` Please use it in order to fulfill the project requirements 👍